spec/airbrake_spec.rb

```diff
@@ -6,7 +6,7 @@
   end
 
   before do
-    described_class.configure do |c|
+    @notifier = described_class.configure do |c|
```

Comment:
`let` or `subject` cache their values, don't they? Here we need to reset the notifier to make sure that we always use a fresh one.